aim/cli/convert/processors/wandb.py

```diff
@@ -0,0 +1,65 @@
+import click
+from aim import Run, Image, Text, Audio
+
+
+def parse_wandb_logs(repo_inst, entity, project, run_id):
+    try:
+        import wandb
+    except ImportError:
+        click.echo(
+            'Could not process wandb logs - failed to import "wandb" module.', err=True
+        )
+        return
+
+    client = wandb.Api()
+
+    if run_id is None:
+        # process all runs
+        runs = client.runs(entity + "/" + project)
+    else:
+        try:
+            # get the run by run_id
+            run = client.run(f"{entity}/{project}/{run_id}")
+        except:
+            click.echo(f'Could not find run with id "{run_id}"', err=True)
+            return
+        runs = (run,)
+
+    for run in runs:
+        if run.config.items():
+            aim_run = Run(
+                repo=repo_inst,
+                system_tracking_interval=None,
+                experiment=project
+            )
+            aim_run.name = run.id
+            aim_run.description = run.notes
+
+            # Collect params & tags
+            aim_run['params'] = run.config
+            aim_run['tags'] = run.tags
```

Comment:
`aim_run` object has its own tag objects associated, so maybe use that instead of setting as a param?